GA/multi_point_problem_definition.py

```
@@ -0,0 +1,492 @@
+            The index of the operating condition in the multi_oper dictionary. 
+            This is used to extract the correct RPS from the blading dictionary.
+
+        Returns
+        -------
+        None
+        """
+
+        # Compute / update the rotational rates in the blading dictionaries
+        self.ComputeOmega(idx=oper_idx)
+        
+        with open(self._tflow_file_path, "r+") as file:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)